model/post.go

```diff
@@ -665,3 +665,7 @@ func RewriteImageURLs(message string, f func(string) string) string {
 
 	return string(result)
 }
+
+func (p *Post) IsFromOAuthBot() bool {
```

Comment:
This is causing the linter checks to fail cause of an inconsistency in receiver naming. Although `p` is the right choice we are currently using the letter `o` in all the other occurrences. We could either take this time to fix them all if you wish or you can just use `o` to make it pass.